services/migration_service/migration_files/20210715184700_add_flow_run_idx.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE INDEX IF NOT EXISTS tasks_v3_run_id_flow_id_idx ON tasks_v3 USING btree (flow_id, run_id, step_name, task_name);
+CREATE INDEX IF NOT EXISTS artifact_v3_run_id_flow_id_idx ON artifact_v3 USING btree (flow_id, run_id, step_name, task_id, attempt_id, name);
```

Comment:
Note that there is already an index on `(flow_id, **run_number**, step_name, task_name)` for `tasks_v3` and an index on `(flow_id, **run_number**, step_name, task_id, attempt_id, name)` for the artifact table. This just adds a pair of indexes for run_id